chcemvediet/templates/main/homepage/casestudy/main.html

```diff
@@ -0,0 +1,16 @@
+{# vim: set filetype=htmldjango shiftwidth=2 :#}
+{% load trans from i18n %}
+
+{% comment %}
+ %
+ % Context:
+ %  -- None
+ %
+{% endcomment %}
+
+<h2 id="{% trans 'main:homepage:casestudies:anchor' %}" class="text-center">
+  {% trans 'main:homepage:casestudies:heading' %}
```

Comment:
Niekde pouzivas nazov `casestudy` a niekde `casestudies`. Pls zjednot to a vsade pouzivaj rovnaky nazov.

Reply:
vdaka, chcel som pouzit vsade `casestudy` v jednotnom cislo, na prekladove stringy som zabudol :ok: